scipy/stats/_continuous_distns.py

```diff
@@ -4220,10 +4220,6 @@ def fit(self, data, *args, **kwds):
 
 
 def _check_fit_input_parameters(data, args, kwds, fixed_param):
-    if len(args) > 0:
```

Comment:
Yeah, do you remember why we had this before? You're taking it out because `args` can contain the guesses/initial valuesof `b`, `loc`, and `scale` used for optimization, right?

Reply:
I have taken a closer look and according to `rv_continuous.fit`

>  arg1, arg2, arg3,…floats, optional
> -  Starting value(s) for any shape-characterizing arguments (those not provided will be determined by a call to _fitstart(data)). No default value.
> 
> 
>  kwds: floats, optional
> -  loc: initial guess of the distribution’s location parameter.
> - scale: initial guess of the distribution’s scale parameter.
> - Special keyword arguments are recognized as holding certain parameters fixed: ...

So, it looks positional arguments are supposed to be guesses for any applicable shapes. 

Laplace, the reason we wrote this, had no shapes, but here we do, so we could instead change it to check that 

`if len(args) > len(self.shapes)`

Which is extendible to other distributions to use as well.